README.md

```diff
@@ -492,9 +472,8 @@ window.env = {
   ADMIN_GROUP: '<The admin group you would like LISA to check the JWT token for>',
   CUSTOM_SCOPES:[<add your optional list of custom scopes to pull groups from your IdP here>],
   // Alternatively you can set this to be your REST api elb endpoint
-  RESTAPI_URI: 'http://localhost:8080/',
+  API_BASE_URL: '<API GW session endpoint>',
```

Comment:
Do we need to add a note to say that this also includes the stage name? maybe give an example of what that looks like?

```
https://${deployment_id}.execute-api.${regional_domain}/${deployment_stage}
```